spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/repository/query/SpannerStatementQueryExecutor.java

```diff
@@ -582,12 +582,19 @@ else if (part.shouldIgnoreCase() != IgnoreCaseType.NEVER) {
 		}
 	}
 
-	private static void buildLimit(PartTree tree, StringBuilder stringBuilder) {
+	private static void buildLimit(PartTree tree, StringBuilder stringBuilder, ParameterAccessor params) {
```

Comment:
since `Pageable` is the one containing the limit info, I would just pass in `Pageable` instead of the full `ParamAccessor` here.